relay-server/src/services/health_check.rs

```diff
@@ -22,15 +23,29 @@ pub enum IsHealthy {
     Readiness,
 }
 
+impl IsHealthy {
+    pub fn variant(&self) -> &str {
+        match self {
+            IsHealthy::Liveness => "liveness",
+            IsHealthy::Readiness => "readiness",
+        }
+    }
+}
+
+pub struct IsHealthyWrapper {
+    pub kind: IsHealthy,
+    pub received: Instant,
```

Comment:
Same here, we already have the metric how long the operation takes and how long we waited for a result, lots of changes for something we already know.